src/main/resources/api/service_openapi.yaml

```diff
@@ -1153,6 +1194,192 @@ components:
         memberEmail:
           type: string
 
+    CreateControlledGoogleBucketRequestBody:
+      description: Payload for requesting a new controlled resource.
+      type: object
+      properties:
+        common:
+          $ref: '#/components/schemas/ControlledResourceCommonFields'
+        googleBucket:
+          $ref: '#/components/schemas/GoogleBucketCreationParameters'
+
+    ControlledResourceCommonFields:
+      type: object
+      required: [ name, cloningInstructions, jobControl, accessScope ]
```

Comment:
At least currently, the async spec expects `jobControl` to be a top-level field in the request. I don't like pushing it down another layer, even if that means a duplicate field on more async request bodies.

Reply:
@ddietterich do you have a preference here? I know we wanted to bundle all the common properties we could.

Reply:
I think it is OK to have the JobControl one level down. My intent in the async spec was to provide an example, but not dictate the request structure.

@zloery Can you say more about why you would prefer it at the top level?

Reply:
I suspect we'll revisit this structure a bit after the schema migration.